grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -1383,10 +1386,19 @@ class QueryService implements Status {
         final String query =
                 ' select * from comment where project_key = :issueId'
         final SQLQuery sqlQuery = session.createSQLQuery(query)
-        final results = sqlQuery.with {
-            addEntity(Comment)
-            setString('issueId', issueId)
-            list()
+        Collection<Comment> results = null
```

Comment:
Drive by comment ... prefer to return an empty collection rather than null. Using null as a return value is generally not a great idea. When I see it - I try to clean it up.

Reply:
Got it!, will keep an eye on it if I see any i'll correct them too.